src/components/Tooltip/index.tsx

```diff
@@ -1,13 +1,16 @@
 import { transparentize } from 'polished'
-import { ReactNode, useCallback, useEffect, useState } from 'react'
+import { ReactNode, useEffect, useState } from 'react'
 import styled from 'styled-components/macro'
 
 import Popover, { PopoverProps } from '../Popover'
 
+const noOp = () => null
```

Comment:
nit, this should probably go into like a shared utils file since there are a couple of other one-off `noop` function definitions

also ive also always seen it spelled `noop` instead of `noOp` 🤔

Reply:
sounds good I added a TODO for now, seems like something I can do in a separate PR for migration across web.